.travis.yml

```diff
@@ -12,8 +12,8 @@ install:
 script:
   - set -e
   - ./scripts/check-pristine-state package-lock.json
+  - (set -o pipefail; npm test |& tee test.log);
```

Comment:
Just to confirm, `|&` is used to merge std err so that `tee` gets that as well as the std output, right?

Reply:
Exactly, this way both output messages (std out and std err) are send to the log file and will still be also in the build output. Also the exit code is preserved, so if the test fail for example, the build will fail anyway.